daemon/CryptoImpl/crypto_p.h

```diff
@@ -586,6 +582,7 @@ class CryptoRequestQueue : public Sailfish::Secrets::Daemon::ApiImpl::RequestQue
     bool unlockPlugin(const QString &pluginName, const QByteArray &lockCode);
     bool setLockCodePlugin(const QString &pluginName, const QByteArray &oldCode, const QByteArray &newCode);
 
+    void handleCancelation(Sailfish::Secrets::Daemon::ApiImpl::RequestQueue::RequestData *request) Q_DECL_OVERRIDE;
```

Comment:
maybe handleCanceledRequest() for symmetry with handleFinishedRequest()?  or maybe not, if a canceled request then transitions to finished state.  hrm.

Reply:
Indeed, cancelation is not a request. As implemented now, it's an external event that requires the request queue to update itself and possibly notify the plugin having current request pending to cancel them.